apps/cowswap-frontend/tsconfig.json

```diff
@@ -31,7 +31,8 @@
       "@cowprotocol/analytics": ["../../../libs/analytics/src/index.ts"],
       "@cowprotocol/permit-utils": ["../../../libs/permit-utils/src/index.ts"],
       "@cowprotocol/multicall": ["../../../libs/multicall/src/index.ts"],
-      "@cowprotocol/balances-and-allowances": ["../../../libs/balances-and-allowances/src/index.ts"]
+      "@cowprotocol/balances-and-allowances": ["../../../libs/balances-and-allowances/src/index.ts"],
+      "@cowprotocol/bridge": ["../../../libs/bridge/src/index.ts"]
```

Comment:
I think it still makes sense to keep in only in Explorer app.
The most of the types are already in SDK and we will use it from there:
https://github.com/cowprotocol/cow-sdk/blob/main/src/bridging/types.ts